maptivist/src/main/webapp/index.html

```diff
@@ -8,8 +8,8 @@
     <script src="https://www.gstatic.com/firebasejs/7.16.1/firebase-auth.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
     <script src="script.js"></script>
```

Comment:
Is this line now redundant with Line 12?

Reply:
Yes it is I think I merged it incorrectly